configure.ac

```diff
@@ -74,6 +74,12 @@ fi
 AM_CONDITIONAL(ENABLE_TESTS_GLES,
     [test "x$enable_tests_gles" = "xyes"])
 
+HAVE_GTEST="no"
+ifdef([GTEST_LIB_CHECK],
```

Comment:
I believe we should not add gtest to the build of libyami.  It would be better if we add a README file with instructions on how to install gtest for the user and make sure that it is only available when debug is enabled.  If debug is not enabled we should not bother.